src/components/editor/metaform-editor-right-drawer-feature.tsx

```diff
@@ -0,0 +1,569 @@
+          fullWidth
+          type="number"
+          sx={{ height: "50px" }}
+          label={`${title} min value`}
+          value={ min }
+          onChange={ event => updateSliderValue(event.target.value, "min") }
+        />
+        <TextField
+          fullWidth
+          type="number"
+          sx={{ height: "50px" }}
+          label={`${title} max value`}
+          value={ max }
+          onChange={ event => updateSliderValue(event.target.value, "max") }
+        />
+      </>
+    );
+  };
+
+  /**
+   * Render options if selected field type is radio, checklist or select
+   */
+  const renderFieldOptions = () => {
+    if (sectionIndex !== undefined && fieldIndex !== undefined) {
+      return (
+        <>
+          { pendingForm.sections![sectionIndex].fields![fieldIndex].options!.map((field, index) => {
+            return (
+              <Stack spacing={ 2 } direction="row">
+                <TextField
+                  sx={{ height: "50px" }}
+                  value={ field.text }
+                  label={ index }
+                  onChange={ event => updateOptionText({
+                    ...field,
+                    name: slugify(event.target.value),
+                    text: event.target.value
+                  }, index)}
+                />
+                <Button
+                  variant="outlined"
+                  color="error"
+                  sx={{
+                    alignContent: "center",
+                    height: "40px"
+                  }}
+                  value={ index }
+                  onClick={ event => deleteFieldOptions(event, index) }
+                >
+                  <DeleteIcon
+                    color="error"
+                    sx={{
+                      display: "flex",
+                      alignItems: "center",
+                      justifyContent: "center"
+                    }}
+                  />
+                </Button>
+              </Stack>
+            );
+          })}
+          <Button
+            fullWidth
+            sx={{
+              height: "50px"
+            }}
+            onClick={ addNewFieldOption }
+          >
+            { strings.draftEditorScreen.editor.features.addSelectionField }
+          </Button>
+        </>
+      );
+    }
+  };
+  
+  /**
+   * Render html editor
+   * 
+   * @param selectedField selected field
+   */
+  const renderHtmlEditor = (selectedField: MetaformField) => {
+    const { html } = selectedField;
+    return (
+      <TextField
+        fullWidth
+        placeholder={ strings.draftEditorScreen.editor.features.addCustomHtml }
+        multiline
+        rows={ 4 }
+        value={ html }
+        onChange={ event => updateCustomHtml({ ...selectedField, html: event.target.value }) }
+      />
+    );
+  };
+
+  /**
+   * Update column title value
+   * 
+   * @param tableColumn MetaformTableColumn where we are changing title
+   * @param index index value of current column title
```

Comment:
Column index? Rename also the parameter. Check for similar comments in whole file